File: ARCHIVE/2018-03-28-2100/1046.py
arr = raw_input ()
arr = arr.split (' ')

a = int (arr[0])
b = int (arr[1])

if (a == b):
    ans = 24
elif (a > b):
    ans = (24-a)+b
else:
    ans = b-a

print "O JOGO DUROU "+ str (ans) + " HORA(S)"

File: ARCHIVE/uri/1-Beginner/1046.c
#include <stdio.h>

int main(){
	
	int st, end, dur;
	scanf("%d%d", &st, &end);

	if (end > st){

		dur = end - st;

	}
	else{

		dur = (end + 24) - st;
	}

	printf("O JOGO DUROU %d HORA(S)\n", dur);

	return 0;
}